### PR TITLE
KRACOEUS-8506: Fixing JTA rollback exception

### PR DESCRIPTION
**Pull request by gathreya**

### CODE REVIEW

File: coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/person/KeyPersonnelServiceImpl.java [hunk 1]
```diff
@@ -94,6 +94,15 @@ public  void setProposalPersonService (ProposalPersonService proposalPersonServi
      * @param document
      */
     public void populateDocument(ProposalDevelopmentDocument document) {
+        populateCreditSplit(document);
+
+        /* check for new certification questions */
+        for (ProposalPerson person : document.getDevelopmentProposal().getProposalPersons()) {
+            getYnqService().getPersonYNQ(person, document);
```
Comment (blackcathacker) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/person/KeyPersonnelServiceImpl.java]: So this was causing a rollback exception? Why?

Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/person/KeyPersonnelServiceImpl.java]: so in getPersonYNQ, we were changing YNQs which is apprently now causing a concurrentModification exception. In this navigation to the credit split tab situation, we do not really need to do this YNQ check anyway, just populate the credit split so I just do not do that anymore. Like I said, adding the @Transactional code in seems to have started this for some reason and I do not know why.

**Comment by blackcathacker** [comment]
Looks like this is causing a compile exception in the unit tests.

**Comment by gathreya** [comment]
oops, missed the second adapter.

**Comment by gathreya** [comment]
test this please.

**Pull request merged**